doc/api/deprecations.md

```diff
@@ -999,6 +999,18 @@ Type: Documentation-only
 The option `produceCachedData` has been deprecated. Use
 [`script.createCachedData()`][] instead.
 
+<a id="DEPXXXX"></a>
+### DEPXXXX: dgram private APIs
+
+Type: Runtime
+
+The `dgram` API previously contained a number of underscored APIs that were
```

Comment:
Nit: remove "a number of" or even "a number of underscored"?

Reply:
May be the below one : 

> The `dgram` module previously contained several APIs that were never meant to be accessed outside node core. These APIs include :

Reply:
> The `dgram` module previously contained several APIs that were never meant to be accessed outside node core. These APIs include :

@antsmartian That seems better to me, but a few nits:

* `node core` -> `Node.js core` (The project is always referred to as `Node.js`. The executable is `node`.)

* Maybe get rid of "These APIs include" because that suggests it's an incomplete list. Just put a colon after `Node.js core` and list the functions there?